routes/defaults.js

```diff
@@ -23,38 +26,55 @@ module.exports = function (log, P, db, error) {
           reply(
             {
               version: version,
-              commit: commitHash
+              commit: commitHash,
+              dbVer: dbVer
             }
           )
         }
 
-        // if we already have the commitHash, send the reply and return
-        if (commitHash) {
-          return sendReply()
-        }
+        // Use async parallel to carry out these various version collection tasks.
+        // By using this async structure, as many versions of supporting services
+        // can be collected as fast as node can GET and process them.
+        async.parallel([
+          function(callback) {
+            // Get the version from fxa-auth-db-mysql server
+            httprequest.get('http://127.0.0.1:8000', function (error, response, body) {
```

Comment:
@jamonation I think you'll have to do something like:

```js
var config = require('../config').root();
```
then you should have access via `config.httpd.url`.